app/src/main/AndroidManifest.xml

```diff
@@ -20,7 +20,8 @@
         tools:ignore="LockedOrientationActivity">
```

Comment:
Why do we have something like this in the manifest?

Reply:
Without it `Expecting android:screenOrientation="unspecified" or "fullSensor" for this activity so the user can use the application in any orientation and provide a great experience on Chrome OS devices` warning comes up (draws line yellow backgrounded).